handbook.md

```diff
@@ -742,8 +743,8 @@ The goal of Atlantian Youth Rapier is to recreate pre-17th century fencing.  All
         #. Fighters with a valid authorization from another kingdom who move into Atlantia may trade in for an Atlantian authorization card. See the Policies of the MoL for the procedure.
     #. The Kingdom Earl Marshal or Deputy Earl Marshal for Rapier Combat shall warrant a select group of marshals for newly approved authorization forms.  Future authorizations shall follow the standard authorization policies and procedures.
 #. Rapier Combat Weapons and Armor Standards
-    #. Atlantia follows the Corporate Rules for Rapier in the Society for Creative Anachronism, Incorporated (found at the Society Marshals Web Site at [(http://www.sca.org/officers/marshal/combat/rapier/index.html),](http://www.sca.org/officers/marshal/combat/rapier/index.html), with the following modifications:
-        #. No spears are allowed.
+    #. Atlantia follows the Corporate Rules for Rapier in the Society for Creative Anachronism, Incorporated (found at the Society Marshals Web Site at [(http://www.sca.org/officers/marshal/combat/rapier/index.html)](http://www.sca.org/officers/marshal/combat/rapier/index.html), with the following modifications:
+        #. Only rubber-headed spears are allowed.
```

Comment:
While I concur with the rubber-headed spear.  Now with the new rules steel spears are still there.  Is there thought being given to including them. I think at this point it may be worthwhile revisiting them.  Based on the following.  Armor standards for hands have changed.  Awareness has increased.  Finally, may just be time for the look.  Atlantia used to be a leader in these kind of things, I think we are the ones that can do it correctly.

Reply:
Steel spears will be unapproved by the end of the year by society.  I don't see a point in supporting them for only a few months given COVID-19.

Reply:
Cool and sad, that makes great sense.  They were introduced so poorly we are losing a possible outlet for fun.